packages/vscode-extension/src/language/file-system.ts

```diff
@@ -55,4 +55,11 @@ export class VSCodeFileSystemProvider implements FileSystemProvider {
   deleteFile(_uri: URI): Promise<void> {
     throw new Error("Not supported.");
   }
+  async findFilesByGlob(pattern: string): Promise<string[]> {
+    const result = (await this._connection.sendRequest(
+      Messages.FindFilesByGlob,
+      pattern,
+    )) as string[];
+    return (result || []).map((p) => p);
+  }
```

Comment:
Suggestion: The corresponding `onRequest(Messages.FindFilesByGlob)` is not implemented. You can use `vscode.workspace.findFiles` for that.